src/device-registry/bin/www

```diff
@@ -3,258 +3,270 @@
+  altitude: Joi.number().optional(),
+  s1_pm2_5: Joi.number().optional(),
+  battery: Joi.number().optional(),
+  device_humidity: Joi.number().optional(),
+  s1_pm10: Joi.number().optional(),
+  device_temperature: Joi.number().optional(),
+  latitude: Joi.number().precision(5).optional(),
+  pm2_5_raw_value: Joi.number().optional(),
+  pm2_5: Joi.number().optional(),
+  pm10_raw_value: Joi.number().optional(),
+  pm10: Joi.number().optional(),
+  timestamp: Joi.date().iso().required(),
+  device_id: Joi.string().empty("").required(),
+  site_id: Joi.string().required(),
+  device_number: Joi.number().optional(),
+  atmospheric_pressure: Joi.number().optional(),
+  humidity: Joi.number().optional(),
+  temperature: Joi.number().optional(),
+  wind_direction: Joi.number().optional(),
+  wind_gusts: Joi.number().optional(),
+  radiation: Joi.number().optional(),
+  wind_speed: Joi.number().optional(),
+  vapor_pressure: Joi.number().optional(),
+  precipitation: Joi.number().optional(),
+  station_code: Joi.string().empty("").optional(),
+  pm2_5_calibrated_value: Joi.number().optional(),
+  pm10_calibrated_value: Joi.number().optional(),
 }).unknown(true);
 
 const eventsSchema = Joi.array().items(eventSchema);
 
-logObject("KAFKA_BOOTSTRAP_SERVERS", constants.KAFKA_BOOTSTRAP_SERVERS)
+logObject("KAFKA_BOOTSTRAP_SERVERS", constants.KAFKA_BOOTSTRAP_SERVERS);
 
 // Set up the Kafka client
 const kafka = new Kafka({
-    clientId: constants.KAFKA_CLIENT_ID,
-    brokers: constants.KAFKA_BOOTSTRAP_SERVERS
+  clientId: constants.KAFKA_CLIENT_ID,
+  brokers: constants.KAFKA_BOOTSTRAP_SERVERS,
 });
 
 /***
  *  make a log directory, just in case it isn't there.
  */
 
 try {
-    require('fs').mkdirSync('./log');
+  require("fs").mkdirSync("./log");
 } catch (e) {
-    if (e.code != 'EEXIST') {
-        console.error("Could not set up log directory, error was: ", e);
-        process.exit(1);
-    }
+  if (e.code != "EEXIST") {
+    console.error("Could not set up log directory, error was: ", e);
+    process.exit(1);
+  }
 }
 
 /****
  * pass the app instance to the middleware configuration
  */
 try {
-    middlewareConfig(app);
+  middlewareConfig(app);
 } catch (e) {
-    logElement("middleware error", e.message)
+  logElement("middleware error", e.message);
 }
 
 // Set up the Kafka consumer
 const consumer = kafka.consumer({ groupId: constants.UNIQUE_CONSUMER_GROUP });
 
 // Function to start consuming KAFKA messages
-const run = async() => {
-    await consumer.connect();
-    await consumer.subscribe({ topic: constants.HOURLY_MEASUREMENTS_TOPIC, fromBeginning: true });
-    await consumer.run({
-        eachMessage: async({ topic, partition, message }) => {
-            if (isEmpty(message)) {
-                logger.error(
-                    `KAFKA: the sent message in undefined --- ${JSON.stringify(
-                        message
+const run = async () => {
+  await consumer.connect();
+  await consumer.subscribe({
```

Comment:
I do not understand why the PR is making styling changes to files/microservices that are not within the scope of the PR? @123MwanjeMike

Reply:
Hi @Baalmart 
Those changes were made by the linter in the pre-commit hook.

Reply:
I've removed all the linter changes @Baalmart